### PR TITLE
XCode15, NDK 25, up v8 and jsc

### LINKED ISSUE #51: Android NDK25
Next RN version will need NDK25
quick test shows issues with V8Inspector
https://github.com/BabylonJS/BabylonNative/pull/1279
Also, add the option to switch NDK version in pipeline

### PR DESCRIPTION
**Pull request by CedricGuillemet**
fixes #51

### CODE REVIEW

File: .github/jobs/ios.yml [hunk 1]
```diff
@@ -1,14 +1,20 @@
+parameters:
+  name: ''
+  vmImage: ''
+  xCodeVersion: ''
+  simulator: ''
+
 jobs:
-- job: iOS
+- job: ${{ parameters.name }}
```
Comment (bghgary) [inline .github/jobs/ios.yml:8]: nit: I've been using no spaces in JsRuntimeHost. (A bunch of places)
```suggestion
- job: ${{parameters.name}}
```

File: Core/AppRuntime/V8Inspector/Source/V8InspectorTCP.h [hunk 2]
```diff
@@ -3,6 +3,13 @@
 // This code is based on the old node inspector implementation. See NOTICE.md for Node.js' project license details
 #pragma once
 
+// 'allocator<void>' is deprecated 
+// Android with NDK25
+// 23/10/04 : allocator still present in latest asio main branch
+#if defined(__clang__)
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+#endif
```
Comment (bghgary) [inline Core/AppRuntime/V8Inspector/Source/V8InspectorTCP.h:11]: Maybe we should submit a fix to `asio`. @ryantrem and I did some simple replacements locally to fix this.

**Review by bghgary — APPROVED** [review]

**Comment by CedricGuillemet** [comment]
https://github.com/BabylonJS/JsRuntimeHost/issues/66

**Comment by CedricGuillemet** [comment]
Closing as these changes are part of https://github.com/BabylonJS/JsRuntimeHost/pull/70

**Pull request closed without merging**